collect_app/src/main/java/org/odk/collect/android/provider/FormsProvider.java

```diff
@@ -113,25 +112,46 @@ public Cursor query(@NonNull Uri uri, String[] projection, String selection, Str
         Cursor cursor;
         switch (URI_MATCHER.match(uri)) {
             case FORMS:
-                cursor = databaseQuery(projectId, projection, selection, selectionArgs, sortOrder, null, PROJECTION_MAP);
+                cursor = databaseQuery(projectId, projection, selection, selectionArgs, sortOrder, null, null);
                 cursor.setNotificationUri(getContext().getContentResolver(), FormsProviderAPI.getUri(projectId));
                 break;
 
             case NEWEST_FORMS_BY_FORM_ID:
-                Collection<String> allColumns = new HashSet<>(PROJECTION_MAP.values());
-                allColumns.remove(DATE);
-                allColumns.add("MAX(date)");
-
-                Map<String, String> maxDateProjectionMap = new HashMap<>(PROJECTION_MAP);
-                maxDateProjectionMap.put("MAX(date)", DATE);
+                Set<String> maxDateColumns = new HashSet<>();
+                maxDateColumns.add(_ID);
```

Comment:
And there's no way to get a projection map for all columns, is there? Or a set of all column names or something that would keep us from having to ever touch this again?

Reply:
Not right now. We could reorg the way the columns are defined to set that up for ourselves, but given this whole way of accessing data is deprecated, I'm not sure if that makes sense right now.